df2gspread/gspread2df.py

```diff
@@ -82,7 +82,9 @@ def download(gfile, wks_name=None, col_names=False, row_names=False,
     raw_data = wks.get_all_values()
 
     if not raw_data:
```

Comment:
`raw_data` is not a pandas object, so I'd go with python `ValueError` here

Reply:
Very valid point. Amended and pushed.